aws_list_all/apply_filter.py

```diff
@@ -0,0 +1,167 @@
+        filter = AppstreamImagesFilter()
+        filter.execute(listing, response)
+
+    if not('cloudsearch' in unfilterList):
+        filter = CloudsearchFilter()
+        filter.execute(listing, response)
+
+    if not('cloudTrail' in unfilterList):
+        filter = CloudTrailFilter()
+        filter.execute(listing, response)
+
+    if not('cloudWatch' in unfilterList):
+        filter = CloudWatchFilter()
+        filter.execute(listing, response)
+
+    if not('iamPolicies' in unfilterList):
+        filter = IAMPoliciesFilter()
+        filter.execute(listing, response)
+
+    if not('s3Owner' in unfilterList):
+        filter = S3OwnerFilter()
+        filter.execute(listing, response)
+
+    if not('ecsClustersFailure' in unfilterList):
+        filter = ECSClustersFailureFilter()
+        filter.execute(listing, response)
+
+    if not('pinpointGetApps' in unfilterList):
+        filter = PinpointGetAppsFilter()
+        filter.execute(listing, response)
+
+    if not('ssmBaselines' in unfilterList):
+        filter = SSMBaselinesFilter()
+        filter.execute(listing, response)
+
+    if not('dbSecurityGroups' in unfilterList):
+        filter = DBSecurityGroupsFilter()
+        filter.execute(listing, response)
+
+    if not('dbParameterGroups' in unfilterList):
+        filter = DBParameterGroupsFilter()
+        filter.execute(listing, response)
+
+    if not('dbClusterParameterGroups' in unfilterList):
+        filter = DBClusterParameterGroupsFilter()
+        filter.execute(listing, response)
+
+    if not('dbOptionGroups' in unfilterList):
+        filter = DBOptionGroupsFilter()
+        filter.execute(listing, response)
+
+    if not('ec2VPC' in unfilterList):
+        filter = EC2VPCFilter()
+        filter.execute(listing, response)
+
+    if not('ec2Subnets' in unfilterList):
+        filter = EC2SubnetsFilter()
+        filter.execute(listing, response)
+
+    if not('ec2SecurityGroups' in unfilterList):
+        filter = EC2SecurityGroupsFilter()
+        filter.execute(listing, response)
+
+    if not('ec2RouteTables' in unfilterList):
+        filter = EC2RouteTablesFilter()
+        filter.execute(listing, response)
+
+    if not('ec2NetworkAcls' in unfilterList):
+        filter = EC2NetworkAclsFilter()
+        filter.execute(listing, response) 
+
+    if not('ec2FpgaImages' in unfilterList):
+        filter = EC2FpgaImagesFilter()
+        filter.execute(listing, response) 
+
+    if not('workmailDeletedOrganizations' in unfilterList):
+        filter = WorkmailDeletedOrganizationsFilter()
+        filter.execute(listing, response) 
+
+    if not('elasticacheSubnetGroups' in unfilterList):
+        filter = ElasticacheSubnetGroupsFilter()
+        filter.execute(listing, response)    
+
+    filter = NextTokenFilter(apply_complete)
+    filter.execute(listing, response)
+    apply_complete = getattr(filter, 'complete')
+
+    return apply_complete
+
+def convert_unfilterList(unfilter):
```

Comment:
It's not needed, but a short version would be `return unfilter if isinstance(unfilter, list) else [unfilter]`